code/datums/quirks/negative.dm

```diff
@@ -11,44 +11,188 @@
 	medical_record_text = "Patient scans indicate severe and chronic back pain."
 	hardcore_value = 4
 	var/datum/weakref/backpack
-
-/datum/quirk/badback/add()
+	var/datum/weakref/left_cane
+	var/datum/weakref/right_cane
+	/// List of objects that should count as canes
+	var/static/list/cane_whitelist = typecacheof(list(
+		/obj/item/cane,
+		/obj/item/staff,
+		/obj/item/godstaff,
+		/obj/item/scythe,
+		/obj/item/pitchfork,
+		/obj/item/gun/magic/staff,
+		/obj/item/nullrod/staff,
+		/obj/item/nullrod/scythe,
+		/obj/item/nullrod/claymore/bostaff,
+		/obj/item/nullrod/pitchfork,
+		/obj/item/nullrod/egyptian,
+		/obj/item/nullrod/spear,
+		/obj/item/spear, // why yes, even the common spear is valid
+		/obj/item/pushbroom,
+		/obj/item/mop,
```

Comment:
Can you turn this type-list into a trait you put on the object? Something like TRAIT_CAINLIKE?

Reply:
That's a great idea.  Initially I was attempting to make it as an element but it looked ugly.  I forgot you could attach traits to items.

Reply:
Applied with https://github.com/tgstation/tgstation/pull/70346/commits/aaf5e8f072631a4f55376ec625aaef2536ef2495